modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -1127,6 +1134,11 @@ public void testBuildTemplateSoyPortlet() throws Exception {
 		File gradleProjectDir = _buildTemplateWithGradle(
 			"soy-portlet", "foo", "--package-name", "com.liferay.test");
 
+		_testContains(
```

Comment:
I think there's another `_testContains` for `build.gradle` in this method, right? Can we merge the 2 to avoid reading the same file twice?